roles/ovirt-repositories/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+ovirt_repositories_repos_backup_path: "/tmp/repo-backup-{{ '%Y-%m-%d-%H:%M:%S' | strftime(ansible_date_time.epoch) }}"
+ovirt_repositories_use_subscription_manager: False
+ovirt_repositories_pool_ids: !!seq []
```

Comment:
Why do we need to use `!!seq []` here?

Reply:
It will probably work without the seq but I followed YAML specification [1] where they define lists like this. 
[1] http://www.yaml.org/spec/1.2/spec.html

Reply:
Sure, but is there a reason to use explicit type here?